src/handlers/imgix.ts

```diff
@@ -6,21 +6,10 @@ import {
   IMGIX_TOKEN as secureURLToken,
 } from 'react-native-dotenv';
 import { Source } from 'react-native-fast-image';
-import { MMKV } from 'react-native-mmkv';
 import parse from 'url-parse';
 import { isCloudinaryStorageLink, signUrl } from '@/handlers/cloudinary';
-import { STORAGE_IDS } from '@/model/mmkv';
 import logger from '@/utils/logger';
 
-export const imgixCacheStorage = new MMKV({
-  id: STORAGE_IDS.IMGIX_CACHE,
```

Comment:
can we add a migration or something to delete the entries for this?

Reply:
Included in #4358, that cool?

Reply:
it's beautiful ser